target/linux/mediatek/filogic/base-files/etc/board.d/01_leds

```diff
@@ -182,6 +182,11 @@ smartrg,sdg-8733a)
 	ucidef_set_led_netdev "wan-orange" "WAN" "mdio-bus:08:orange:wan" "wan" "link_100 link_1000"
 	ucidef_set_led_netdev "wan-white" "WAN" "mdio-bus:08:white:wan" "wan" "link_10000"
 	;;
+snr,cpe-ax2)
+	ucidef_set_led_netdev "ledwanblue" "wan" "blue:wan" "wan" "link tx rx"
+	ucidef_set_led_netdev "led2g" "WLAN2G" "blue:wlan-2ghz" "phy0-ap0" "link tx rx"
+	ucidef_set_led_netdev "led5g" "WLAN5G" "blue:wlan-5ghz" "phy1-ap0" "link tx rx"
```

Comment:
These LEDs already have default triggers in the dts and these configuration can be removed.